### DISCUSSION TITLE
Save output configuration

### DISCUSSION THREAD

**Discussion by mquevill** [TinkerTools/tinker-gpu#160] [General]
Canonical Tinker would print out an output configuration file after a run with `dynamic` that would get appended with the version (file.xyz, file.xyz_2, etc.). I have been running simulations from the `master` branch of Tinker9, and it has not created these output files. While I can extract this last frame from file.arc (regardless of `archive` being present in the keyword file), having it automatically created at the end of the run is something that can be very useful.

Is there a keyword that I am missing to get this printed out? I have tried tweaking a few things, but have not been able to get that output.
 
The following commands create only "ar94.arc" and "ar94.dyn" files.
```
cd example/
tinker9 dynamic ar94 10000 1 1 1
```

**Comment by zhi-wang** [comment]
I think ".xyz_2" file must have been created by another program, probably "minimize". I don't remember that "dynamic" may create an ".xyz_2" file.

**Comment by mquevill** [comment]
Ah yes, you are right - `dynamic` doesn't produce an output .xyz_2 file. While the resulting .dyn file can be used to (re)start a simulation, is there an easy way of extracting the last configuration of an archive file without knowing how many steps were taken?

**Comment by zhi-wang** [comment]
Let's say you have 1000 atoms in your system, how about the following command:

```bash
tail -1002 file.arc > file.xyz_2
```

**Comment by leucinw** [comment]
How about this 

_tail -n \`cat your.xyz | wc -l\` your.arc > your.xyz_2_
